### ISSUE TITLE
about conflict between the paper and this implement

### ISSUE THREAD

**Issue by WeianMao** [Jeff-sjtu/res-loglikelihood-regression#25]
thanks for your great work. i prepare to site your paper. but i meat the mask rcnn's setting issue. in your paper, you said mask rcnn is trained without heatmap pretrain and the trainin schedule is 3X (27000iter). but this is different from the setting you mentioned on another issue  #12  , in which you said you trained the model using 2X and heatmap pretrain. so can you please provide the excat setting of the mask rcnn you used in you paper? thanks a lot!

**Comment by jeffffffli** [comment]
Please refer to https://github.com/Jeff-sjtu/res-loglikelihood-regression/issues/12#issuecomment-956086000.
